remote-playback/state-attribute-changes-when-selecting-device-manual.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <title>
+    Test that the remote playback state changes when selecting a device.
+  </title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/media.js"></script>
+  <body>
+    <div id="prep">
+      <p>Please make sure a device for remote playback is <b>available.</b></p>
+      <button id="prompt-button-prep">Show devices</button>
+      <button id="start-button">Start test</button>
+    </div>
+    <div id="pick-device" style="display: none">
+      <p>
+        Click the button below to prompt for a remote playback device and select
+        one!
+      </p>
+      <button id="prompt-button">Pick device</button>
+    </div>
+  </body>
+  <script>
+    var v = document.createElement("video");
+    v.src = getVideoURI("/media/movie_5");
+
+    var startButton = document.getElementById("start-button");
+    startButton.onclick = function () {
+      var prep = document.getElementById("prep");
+      prep.style.display = "none";
+      var pickDeviceOptions = document.getElementById("pick-device");
+      pickDeviceOptions.style.display = "block";
+    };
+    var promptPrepButton = document.getElementById("prompt-button-prep");
+    promptPrepButton.onclick = function () {
+      v.remote
+        .prompt()
+        .then(() => {})
+        .catch(() => {});
+    };
+    setup({
+      explicit_timeout: true,
+    });
+    async_test((t) => {
+      assert_equals(v.remote.state, "disconnected");
+
+      function callback(available) {
+        //assert_true(available, "Is remote playback device available");
+        promise_test(() => {
+          return v.remote.prompt().then(
+            t.step_func(() => {
+              assert_equals(v.remote.state, "connecting");
+            })
+          );
+        }, "Prompt call resolves");
+      }
```

Comment:
After resolving the prompt() Promise, the browser should also transition the state to "connected" at some point in the future.  However, the spec does not give a deadline for this transition.   Since you're testing the state with the event handlers in the next test, I think there's good test coverage of the "connected" state.

Reply:
Based on this information, I don't really understand what I am supposed to change in the tests.